### DISCUSSION TITLE
antd-pro那边的一堆文档指向umi，结果全报页面未找到

### DISCUSSION THREAD

**Discussion by KashingLiu** [umijs/umi#11955] [General]
这文档到底写了个啥，写了个寂寞

https://pro.ant.design/zh-CN/docs/request 链接中的第一个外链就提示页面不存在。
https://umijs.org/zh-CN/plugins/plugin-request#request

<img width="1327" alt="image" src="https://github.com/umijs/umi/assets/22939122/3d3f27b6-2e87-43d7-b7bc-78d55fd3fdb7">

**Comment by fz6m** [comment]
ant design pro 这个模板已经很长一段时间没有人投入维护了，目前可以通过 clone 或者 cli 生成的方式使用里面的 umi 4 模板，默认使用的是 `@umija/max` ，这是推荐的使用方式。

而目前 ant design pro 的 文档 和 preview 页面都是旧的，也就是 umi 3 的模板，如果你需要使用旧的模板，可以把 url 加一个 `v3.umijs.org` 来访问旧 v3 的文档，但不推荐使用 umi 3 了，因为 umi 5 快要开始了。
